src/Symfony/Component/Validator/Constraints/Url.php

```diff
@@ -24,5 +24,7 @@
 class Url extends Constraint
 {
     public $message = 'This value is not a valid URL.';
+    public $dnsMessage = 'The host could not be resolved.';
```

Comment:
Should this new message be added to `src/Symfony/Component/Validator/Resources/translations/validators.en.xlf` file in order to translate it?